juju/client/connection.py

```diff
@@ -433,6 +433,7 @@ def _exit_tasks():
             max_size=self.max_frame_size,
             server_hostname=server_hostname,
             sock=sock,
+            close_timeout=1,
```

Comment:
I actually don't think we want to change this. If it is something that we want to expose to clients we can, but I'd rather use default timeouts unless we have a real need to change it. (This seems sane, except when you might have poor connectivity, etc.)
Since we know we don't need to change it when things are 'happy' (in my testing I can do the annotation round trip in 400ms and exit success).

Reply:
Yeah I agree about this, let's not change it if we don't have to.